src/shared/libfido2-util.c

```diff
@@ -546,6 +634,33 @@ int fido2_use_hmac_hash(
                         goto finish;
                 }
 
+                r = fido2_is_cred_in_specific_token(path, rp_id, cid, cid_size, pins, required);
+                /* We handle PIN related errors here since we have used PIN in the check.
+                 * If PIN error happens, we can avoid pin retries decreased the second time. */
+                if (IN_SET(r, -ENOANO,        /* → pin required */
+                              -ENOLCK,        /* → pin incorrect */
+                              -EOWNERDEAD)) { /* → uv blocked */
+                        /* If it's not the last device, try next one */
+                        if (i < found - 1)
+                                continue;
+
+                        /* -EOWNERDEAD is returned when uv is blocked. Map it to EAGAIN so users can reinsert
+                         * the token and try again. */
+                        if (r == -EOWNERDEAD)
+                                r = -EAGAIN;
+                        goto finish;
+                } else if (r < 0) {
+                    if (r == -NODEV)
+                        continue;
+
+                    log_error_errno(r, "Failed to determine whether the credential is in the token, try anyway: %m");
```

Comment:
indentation. please follow coding style. multiples of 8 spaces. the .vimrc/elisp/.editorconfig file included in the repo should actually force you into this already